.github/workflows/build_servers_vip.yml

```diff
@@ -1,10 +1,13 @@
-name: Build Servers from a Pull Request
-# build_servers.yml
+name: Build servers in VIP mode
```

Comment:
If we want to build VIP, just include it in `build_servers_gcc`/`build_servers_msbuild`?